stetho-timber/src/main/java/com/facebook/stetho/timber/StethoTree.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.timber;
+
+import android.util.Log;
+import com.facebook.stetho.inspector.console.CLog;
+import com.facebook.stetho.inspector.console.ConsolePeerManager;
+import com.facebook.stetho.inspector.protocol.module.Console;
+import timber.log.Timber;
+
+public class StethoTree extends Timber.Tree {
```

Comment:
Maybe some javadocs explaining usage?  Maybe a one-liner <pre> statement showing how to plant the stetho tree in timber?